packit/utils/bodhi.py

```diff
@@ -9,18 +23,46 @@
 logger = logging.getLogger(__name__)
 
 
+class OurBodhiClient(BodhiClient):
+    """compat layer for bodhi 5 and 6 clients"""
+
+    def __init__(self, fas_username: Optional[str] = None, fas_password: str = None):
```

Comment:
Just a nitpick, the type of `fas_password` should probably also be `Optional[str]`.